.github/workflows/docker-publish.yml

```diff
@@ -1,5 +1,8 @@
 name: Docker
 
+permissions:
+  contents: read
```

Comment:
@psilva-veeam Makes sense. Also, these are the default permissions for the GH token for this repo, these are enforced.